contact.html

```diff
@@ -4,8 +4,16 @@
         
         <title>Doc2Pen|Contact</title>
         
-    <meta charset="utf-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- meta tags -->
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="keywords" content="doc2pen, Doc2Pen, Text-to-handwriting, text to handwriting converter online free, convert text to handwriting app, text to handwriting online free,
```

Comment:
1) Please do not  repeat the same keywords like ```doc2pen``` and ```Doc2Pen```
2) I would not suggest using ```text to handwriting```
3) Have short keywords

Reply:
Sure, I'll do the above changes.